### ISSUE TITLE
[Stretch goal] Create Chart Page

### ISSUE THREAD

**Issue by ecuellog** [ChoralCloud/ChoralWeb#44]
Create a page where user can create charts. This should list all of the user's chorals to choose from which one the chart will be created. (Name: Victoria weather, Choral: [choral selection list])

**Comment by andreirtaylor** [comment]
We should add a section to the choral API doc on the drive about this before we start implementing this. We should define the structure of the expected object which defines a chart.
